tests/foreman/api/test_host.py

```diff
@@ -423,7 +423,6 @@ def test_positive_create_and_update_with_subnet(module_location, module_org, mod
 
 
 @pytest.mark.tier2
-@pytest.mark.on_premises_provisioning
```

Comment:
Do we need to remove this marker?

Reply:
yes, this isn't related to on_premises_provisioning and doesn't require any specific setup